server.js

```diff
@@ -0,0 +1,170 @@
+const express = require('express');
+const session = require('express-session');
+const path = require('path');
+
+const app = express();
+
+const db = require('./db');
+const { models } = require('./db');
+const { Product, User, Guest, Order, OrderProducts } = models;
+
+// Setups for express-sessions
+const TWO_HOURS = 1000 * 60 * 60 * 2;
+
+const SESS_NAME = 'sid';
+const SESS_SECRET = 'BRAVO';
+const SESS_LIFETIME = TWO_HOURS;
+
+const Sequelize = require('sequelize');
+
+//ISSUES WITH WEBPACK - COMMENTING OUT THE BELOW
+app.use(express.json());
+app.use('/assets', express.static(path.join(__dirname, 'assets')));
+
+app.use(
+  session({
+    name: SESS_NAME,
+    resave: false,
+    saveUninitialized: false,
+    secret: SESS_SECRET,
+    cookie: {
+      maxAge: SESS_LIFETIME,
+      sameSite: true
+    }
+  })
+);
+
+const port = process.env.PORT || 3000;
```

Comment:
Inconsistent constant naming, you name things up above `SESS_NAME` when they are constants, yet `port` is lower case.